tests/console/verify_firewall_ssh.pm

```diff
@@ -0,0 +1,22 @@
+# SUSE's openQA tests
+#
+# Copyright © 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+#
+# Summary: Verification of ssh service, allowed by firewalld
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use strict;
+use warnings;
+use parent 'y2_module_consoletest';
+use testapi;
+
+sub run {
+    validate_script_output("firewall-offline-cmd --zone=external --list-services", sub { /ssh/ });
```

Comment:
I wonder why do we check ssh, or only ssh here? As http and https rules were validated before, as we have apache services added explicitly to the profile.

Reply:
Actually, I wanted to ask about this, because the previous script was supposed to validate http(s) but still, the command output would be ssh and there was no error. https://openqa.suse.de/tests/5329100#step/autoyast_verify/8

Reply:
Error is there, but in the script it just echo the text. I'm not sure what exactly is expected, might be that services are configured for other zone. But in the autoyast profile, I clearly see:
```
<firewall>
<FW_CONFIGURATIONS_EXT>apache2 apache2-ssl</FW_CONFIGURATIONS_EXT>
<FW_LOG_DROP_ALL>no</FW_LOG_DROP_ALL>
<FW_SERVICES_EXT_TCP>8080</FW_SERVICES_EXT_TCP>
<FW_SERVICES_EXT_UDP>9090</FW_SERVICES_EXT_UDP>
<enable_firewall config:type="boolean">false</enable_firewall>
<start_firewall config:type="boolean">false</start_firewall>
</firewall>
```

Reply:
Good catch. Seems that the autoyast syntax is for SLE12 that uses SuSEfirewall2 . I will investigate furtherly and change what needs to be changed.

Reply:
I am confuzed now because the syntax is supported but deprecated. https://documentation.suse.com/sles/15-SP2/html/SLES-all/appendix-ay-12vs15.html#sec-ay-12vs15-firewall 
but, there is the statement "Configuration options from SuSEfirewall2 that are no longer available either have no equivalent mapping in firewalld or will be supported in future releases of SUSE Linux Enterprise Server." 
I can't  find option "FW_CONFIGURATIONS_EXT" for firewalld.

Reply:
We can also adjust the profile. Most of the options should still work for both.